src/map/trait.h

```diff
@@ -139,7 +139,11 @@ enum TRAITTYPE
     TRAIT_PRIMEVAL_ZEAL       = 120,
     TRAIT_INSPIRATION         = 121,
     TRAIT_SLEIGHT_OF_SWORD    = 122,
-    TRAIT_DAKEN               = 123
+    TRAIT_DAKEN               = 123,
+    TRAIT_SUPERIOR            = 124, 

```

Comment:
The next one is magic accuracy bonus. I mean... I can go ask a friend with
superior unlocked, but I don't see why they'd randomly have superior
between other traits. Besides, having superior as a trait makes total
sense. It's the exact same thing as dual wield